src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -0,0 +1,632 @@
+      (availableOffer) => availableOffer.provider === selectedOffer?.value
+    );
+    resetTransactionBlockFieldValidationError(transactionBlockId, 'amount');
+    if (receiverAddress && !isValidEthereumAddress(receiverAddress)) {
+      setTransactionBlockFieldValidationError(
+        transactionBlockId,
+        'receiverAddress',
+        'Invalid receiver address'
+      );
+      return;
+    }
+    resetTransactionBlockFieldValidationError(
+      transactionBlockId,
+      'receiverAddress'
+    );
+    setTransactionBlockValues(transactionBlockId, {
+      hasEnoughPLR,
+      fromChainId: selectedFromNetwork?.chainId ?? undefined,
+      toAsset: hasEnoughPLR ? plrDaoMemberNFT : plrDaoAsset,
+      fromAsset: selectedFromAsset ?? undefined,
+      amount,
+      offer,
+      accountType: selectedAccountType,
+      receiverAddress: receiverAddress ?? undefined,
+    });
+  }, [
+    selectedFromNetwork,
+    selectedFromAsset,
+    amount,
+    selectedOffer,
+    availableOffers,
+    selectedAccountType,
+    receiverAddress,
+  ]);
+
+  const RenderOption = (option: SelectOption) => {
+    const availableOffer = availableOffers?.find(
+      (offer) => offer.provider === option.value
+    );
+    const valueToReceive =
+      availableOffer &&
+      formatAmountDisplay(
+        ethers.utils.formatUnits(
+          availableOffer.receiveAmount,
+          plrDaoAsset.decimals
+        )
+      );
+    return (
+      <OfferDetails>
+        <RoundedImage title={option.title} url={option.iconUrl} size={24} />
+        <div>
+          <Text size={12} marginBottom={2} medium block>
+            {option.title}
+          </Text>
+          {!!valueToReceive && (
+            <Text size={16} medium>
+              {valueToReceive} {plrDaoAsset.symbol}
+            </Text>
+          )}
+        </div>
+      </OfferDetails>
+    );
+  };
+
+  return !isNFTMember ? (
+    <>
+      <Title>Stake into Pillar DAO</Title>
+      <Container>
+        <Text style={{ fontSize: '16px' }}>
+          To become DAO member, you need to stake <Value>10,000 PLR</Value> tokens on Polygon.
+        </Text>
+        <HorizontalLine></HorizontalLine>
+        {
+          <Text style={{ fontSize: '14px' }}>
+            You have{' '}
+            {hasEnoughPLR ? (
+              <Value>{`${formatAmountDisplay(totalKeyBasedPLRTokens + totalSmartWalletPLRTokens)}`} PLR</Value>
+            ) : (
+              <Total>{`${formatAmountDisplay(totalKeyBasedPLRTokens + totalSmartWalletPLRTokens)}`} PLR</Total>
+            )}
+            {' tokens '}
+            {accounts.length > 0
+              ? `on ${accounts.length == 1 ? `${accounts[0].chainName} chain` : `${accounts.length} chains `} on ${
+                  totalKeyBasedPLRTokens > 0 && totalSmartWalletPLRTokens > 0
+                    ? '2 wallets'
+                    : totalKeyBasedPLRTokens > 0
+                    ? 'Key Based'
+                    : 'Smart Wallet'
```

Comment:
please remove multiple ternary as other component or put in memo, but this cannot stay here